tests/ops/integration_tests/saas/test_amplitude_task.py

```diff
@@ -171,6 +171,7 @@ async def test_amplitude_access_request_task(
     assert v[f"{dataset_name}:user_details"][0]["user_id"] == amplitude_identity_email
 
 
+@pytest.mark.skip(reason="Temporarily disabled test")
```

Comment:
TODO for later, the data for Amplitude wasn't settling in a reasonable amount of time.